package.json

```diff
@@ -40,6 +40,7 @@
 		"@tailwindcss/aspect-ratio": "^0.2.1",
 		"@tailwindcss/line-clamp": "^0.2.1",
 		"@tailwindcss/typography": "^0.4.1",
-		"rive-js": "^0.7.33"
+		"rive-js": "^0.7.33",
+		"yarn": "^1.22.19"
```

Comment:
I don't think it makes sense to add an additional dependency. I'd actually prefer if `npm` was used instead of `yarn`. Would you like to make that change?

Reply:
Sure thing, thanks for the feedback!

Reply:
Hey @stnguyen90, I have a small query about this - if we're not adding any dependency to `package.json` in favour of adding instructions to install `npm`, instead of replacing `yarn` with `npm` in `package.json` (_as the general pattern followed by most users is `npm install`, rendering `npm` in `package.json` reduntant_), we should perhaps modify `README.md` and add instructions to install `npm` instead, or create a shell script that (A) installs npm and (B) runs `npm install`, which fetches and installs the currently existing dependencies.

What do you think?

Reply:
Okay, thanks for the clarification! I think this PR is out of scope in that case, if we're not planning to add yarn as a dependency and not adding npm instructions.

What do you think?

Reply:
Sure.

Reply:
Thanks a lot for the feedback @stnguyen90!

I'll close this PR, let's meet in another PR :)